### ISSUE TITLE
CC and BCC labels in write window not localized

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/yam#495]
**Originally by _nbache2@mail.dk_ on 2013-12-29 19:18:27 +0100**

___
## Summary
It seems the two labels for the CC abnd BCC fields in the write window are no longer localized (since moving them from the options tab in 2.8?).
## Steps to reproduce
 1. Translate the strings MSG_WR_CopyTo and MSG_WR_BlindCopyTo, compile and deploy the catalog.
 2. Open a write window either by hitting New Msg or editing an existing one.

## Expected results
That the fields have their translated labels.
## Actual results
The two fields still only have the labels "_CC:" and "_BCC:".
## Notes
Lines 1099-1109 in trunk/src/mui/WriteWindow.c (as of Dec. 29th, 2013), showing the two labels being set to hardcoded values, as opposed to the other two around them:

```
Child, data->LB_TO = Label(tr(MSG_WR_To)),
Child, data->GR_TO = MakeAddressField(&data->ST_TO, tr(MSG_WR_To), MSG_HELP_WR_ST_TO, ABM_TO, data->windowNumber, AFF_ALLOW_MULTI|AFF_EXTERNAL_SHORTCUTS),

Child, data->LB_CC = Label("_CC"),
Child, data->GR_CC = MakeAddressField(&data->ST_CC, tr(MSG_WR_CopyTo), MSG_HELP_WR_ST_CC, ABM_CC, data->windowNumber, AFF_ALLOW_MULTI|AFF_EXTERNAL_SHORTCUTS),

Child, data->LB_BCC = Label("_BCC"),
Child, data->GR_BCC = MakeAddressField(&data->ST_BCC, tr(MSG_WR_BlindCopyTo), MSG_HELP_WR_ST_BCC, ABM_BCC, data->windowNumber, AFF_ALLOW_MULTI|AFF_EXTERNAL_SHORTCUTS),

Child, data->LB_REPLYTO = Label(tr(MSG_WR_ReplyTo)),
Child, data->GR_REPLYTO = MakeAddressField(&data->ST_REPLYTO, tr(MSG_WR_ReplyTo), MSG_HELP_WR_ST_REPLYTO, ABM_REPLYTO, data->windowNumber, AFF_ALLOW_MULTI|AFF_EXTERNAL_SHORTCUTS),
```

Additional note: I am currently finishing the review of my translation for 2.8p1 before moving on to 2.9, so the bug is found in 2.8p1, but the source suggests it is still present in 2.9.

**Comment by jens-maus** [comment]
**Originally on 2013-12-29 19:23:32 +0100**

___
AFAIR that was done intentionally as there seem to be no reason why in another language CC shouldn't be CC or BCC shouldn't be BCC. The only reason to actually add these strings to the catalog is that a translator can choose another shortcut. But then at least for CC he wouldn't have any other choice since there are only "C" characters.

So please outline and explain why you feel the strings should still be added to the catalog.

P.S: Please make sure to generate yourself a proper login for the yam.ch site if you want to be a permanent danish translator of YAM so that we can more easily assign tickets to you in case bug reports come in.

**Comment by jens-maus** [comment]
**Originally by _nbache2@mail.dk_ on 2013-12-29 20:22:05 +0100**

___
In Danish, the normal names for these functions are "Kopi til" and "Blindkopi til". We don't use the somewhat technical and foreign (English/American) abbreviations in normal speech, and many non-technical users will not necessarily know their meaning, but will relate to the more descriptive names, which incidentally have been in use in offices even before e-mail.

My current choice is to combine the two, e.g. "Kopi til (CC)", in these labels - which I believe also used to be the way these labels were in English?

P.S. Login created.

**Comment by jens-maus** [comment]
**Originally on 2013-12-29 20:35:37 +0100**

___
After having rethought about that matter I will assign that ticket for the 2.9p1 milestone as I agree that the current situation seems to be suboptimal and that not all languages really use these technical terms. In addition, it would make choosing a proper/unused hotkey more easy.

**Comment by jens-maus** [comment]
**Originally on 2014-01-09 18:01:09 +0100**

___
In (61ca2fb):
 * mui/WriteWindow.c, locale/YAM.pot: updated the catalog strings for the Cc,
  Bcc and Reply-To adress fields and their corresponding menu items. Now they
  can be properly translated. This closes #495.

**Comment by jens-maus** [comment]
**Originally on 2014-04-09 00:55:37 +0200**

___
**Release Note:**
modified the Cc and BCC text fields to be actually localizable by translators.
